### PR TITLE
Project guess who 

### PR DESCRIPTION
**Pull request by liaback-lowstett**

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -61,7 +64,7 @@ select {
 
 /****** CARD ******/
 .card {
-  width: 135px;
+  width: 120px; /* changed width to fit in screen-view whitout scrolling*/
```
Comment (lars-soderman) [inline code/style.css:67]: 🚀

**Review by lars-soderman — COMMENTED** [review]
Hey Felicia 👋  Really great job with the guess who-game! Everything works like it should, so it is all good. I put in some thoughts and comments in the code. 
Rock on! 💥

File: code/script.js [hunk 2]
```diff
@@ -247,53 +250,86 @@ const generateBoard = () => {
...
+  // Invoke filterCharacters
+  filterCharacters(keep)
 }
 
-// It'll filter the characters array and redraw the game board.
+// Filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
+  let category = currentQuestion.category
+  let attribute = currentQuestion.attribute
+  let value =currentQuestion.value
```
Comment (KarolCislowski) [inline code/script.js:328]: `const { category, attribute, value } = currentQuestion` would do the magic here 🧙🏻

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**